blocks/hero/hero.css

```diff
@@ -1,34 +1,193 @@
-main .hero-container > div {
-  max-width: unset;
-}
-  
-main .hero-container {
+.section.hero-container {
+  background-color: var(--c-gray-lt);
   padding: 0;
 }
 
-.hero {
+.hero .hero-main-container {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
   position: relative;
-  padding: 32px;
-  min-height: 300px;
+  flex-direction: column;
 }
 
-.hero h1 {
-  max-width: 1200px;
-  margin-left: auto;
-  margin-right: auto;
-  color: white;
+/* Hero left container styles */
+.hero .hero-left-container {
+  width: 100%;
+  padding: 64px 24px;
 }
 
-.hero picture {
-  position: absolute;
-  z-index: -1;
-  inset: 0;
-  object-fit: cover;
-  box-sizing: border-box;
+.hero .hero-content {
+  width: 100%;
 }
 
-.hero img {
-  object-fit: cover;
+.hero .hero-content-container h1,
+.hero .hero-content-container h2,
+.hero .hero-content-container h3
+.hero .hero-content-container h4,
+.hero .hero-content-container h5,
+.hero .hero-content-container h6 {
+    background: var(--c-dark-blue-gradient);
+    background-clip: text;
+    -webkit-text-fill-color: transparent; 
+    margin: 0 0 32px;
+}
+
+.hero .hero-content-container p {
+  font-size: 18px;
```

Comment:
please use variable wherever its applicable eg:  --body-font-size-l: calc((18 / 16) * 1rem);

Reply:
Done with the review comments @Vaitheesaran-nagalingam